### PR TITLE
Fixes get_crosslink_committees_at_slot() logic and test

### PR DESCRIPTION
**Pull request by akhila-raju**
## PR Description
Fixes get_crosslink_committees_at_slot() logic and penalizeValidatorDecrementsBadActorAndIncrementsWhistleblower test with accurate mock data.

penalizeValidatorDecrementsBadActorAndIncrementsWhistleblower originally initiated the validator list at size 3. The number of validators required to remove this error is supposed to be at least 64, since split splits the validator list into 64 committees. The issue was that split would return empty lists for the empty committees - and the divide by zero error was due to the first item in the shuffling list in get_crosslink_committees_at_slot being an empty list. Thus, testing with 64 validators should fix the problem, since each committee will be non-empty.

### CODE REVIEW

File: ethereum/datastructures/src/test/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtilTest.java [hunk 1]
```diff
@@ -319,7 +320,74 @@ void penalizeValidatorDecrementsBadActorAndIncrementsWhistleblower() {
     beaconState.setCurrent_shuffling_epoch(Constants.FAR_FUTURE_EPOCH);
     beaconState.setPrevious_shuffling_epoch(Constants.FAR_FUTURE_EPOCH);
     List<Long> latestPenalizedBalances =
-        new ArrayList<>(Arrays.asList(randomLong(), randomLong(), randomLong()));
+        new ArrayList<>(
```
Comment (schroedingerscode) [inline ethereum/datastructures/src/test/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtilTest.java]: I'd suggest using something like https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#nCopies(int,%20T) so this isn't so verbose.

**Review by schroedingerscode — COMMENTED** [review]
I'll let Cem double check this to make sure the changes in `get_genesis_beacon_state` are compliant with the spec. I'd just suggest you use java.util.Collections.nCopies for some of the data generation so the code isn't as verbose.

File: artemis/src/test/java/tech/pegasys/artemis/BeaconNodeIntegrationTest.java [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
   private static ObjectMapper mapper = new ObjectMapper();
 
-  @Test
+  @Disabled
   void testThreeNodes() throws InterruptedException, JsonProcessingException {
```
Comment (jrhea) [inline artemis/src/test/java/tech/pegasys/artemis/BeaconNodeIntegrationTest.java]: Master had a breaking change to this test (was getting Null Pointer exception).  #613 fixed that issue.  This test **should** be able to be enabled now

**Comment by akhila-raju** [comment]
> I'll let Cem double check this to make sure the changes in `get_genesis_beacon_state` are compliant with the spec. I'd just suggest you use java.util.Collections.nCopies for some of the data generation so the code isn't as verbose.

I made the changes so it followed the spec exactly (before it didn't match completely).

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java [hunk 3]
```diff
@@ -145,25 +145,30 @@ public static BeaconStateWithCache get_genesis_beacon_state(
 
     } else if (epoch == next_epoch) {
 
-      long current_committees_per_epoch = get_current_epoch_committee_count(state);
-      committees_per_epoch = get_next_epoch_committee_count(state);
-      shuffling_epoch = next_epoch;
       long epochs_since_last_registry_update =
           current_epoch - state.getValidator_registry_update_epoch();
       if (registry_change) {
+        committees_per_epoch = get_next_epoch_committee_count(state);
```
Comment (jrhea) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java]: These modifications don't seem to match v0.4 of the spec.  Is there a good reason why you had to deviate?  
https://github.com/ethereum/eth2.0-specs/blob/v0.4.0/specs/core/0_beacon-chain.md#get_crosslink_committees_at_slot

Comment (akhila-raju) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java]: I made it to match v0.5, but can switch it back to v0.4

Comment (jrhea) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java]: Ah ok.  Ya let's keep it at v0.4 unless there is a legit functional reason to borrow from v0.5

Comment (akhila-raju) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java]: There is one functional change that was incorrect (line 183), I will revert the other changes

**Review by schroedingerscode — APPROVED** [review]
LGTM given no further comments by @jrhea. (That and make sure to run spotless so Jenkins stops being mad.)

**Review by jrhea — APPROVED** [review]
LGTM (contingent on our nemesis Spotless)

**Pull request merged**